From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Thu, 11 Apr 2019 17:54:09 +0100
Subject: Ubuntu-location-default

# Description: change the default value of the location switch to false
# Description: change the default value of the location switch to false
# which is what the Canonical legal team recommended us to do
#
# Ubuntu: https://bugs.launchpad.net/bugs/1821901
# Upstream: not-needed(?)
#
---
 gnome-initial-setup/pages/privacy/gis-privacy-page.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gnome-initial-setup/pages/privacy/gis-privacy-page.c b/gnome-initial-setup/pages/privacy/gis-privacy-page.c
index a64bd81..9304762 100644
--- a/gnome-initial-setup/pages/privacy/gis-privacy-page.c
+++ b/gnome-initial-setup/pages/privacy/gis-privacy-page.c
@@ -121,7 +121,7 @@ gis_privacy_page_constructed (GObject *object)
   priv->location_settings = g_settings_new ("org.gnome.system.location");
   priv->privacy_settings = g_settings_new ("org.gnome.desktop.privacy");
 
-  gtk_switch_set_active (GTK_SWITCH (priv->location_switch), TRUE);
+  gtk_switch_set_active (GTK_SWITCH (priv->location_switch), FALSE);
   gtk_switch_set_active (GTK_SWITCH (priv->reporting_switch), TRUE);
 
   update_os_data (page);
